### DISCUSSION TITLE
ferretdb for linux/arm/v7

### DISCUSSION THREAD

**Discussion by gocs** [FerretDB/FerretDB#1266] [General]
Is it possible to get linux/arm/v7 builds? or is there another way I can use a mongodb-like database in **raspi 3**?

**Comment by AlekSi** [comment]
@gocs Sorry, I missed that topic. Yes, we will provide them shortly.

**Comment by AlekSi** [comment]
[v0.6.2](https://github.com/ferretdb/FerretDB/pkgs/container/ferretdb/52030390?tag=0.6.2) is out with `linux/arm/v7` support!

**Reply by BoBiene** [reply]
@AlekSi with the release of v2 it looks like you droped the support for arm v7 again. Any plans to readd this to v2?

**Reply by AlekSi** [reply]
We might add it back, but I'm not sure DocumentDB would support that architecture. Is there a use case where FerretDB runs on RPi 3 with PostgreSQL running somewhere else?

**Reply by BoBiene** [reply]
You're right, the RPi 3 uses ARMv8, but since MongoDB 5.0+ requires at least ARMv8.2-A, I'm looking for ARMv8 support in FerretDB to cover that gap. Does FerretDB v2 support Raspberry Pi models >= 3B?

**Reply by AlekSi** [reply]
FerretDB supports arm64; DocumentDB [doesn't yet](https://github.com/microsoft/documentdb/issues/23) (despite the issue title, it is about arm64).
